### ISSUE TITLE
Perception of self-motion

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6803]
Preferred term label:
Perception of self-motion

Definition (free text, please give PubMed ID)
PMID: 26559820

Parent term (use hpo.jax.org/app)
No found because of the specificity of the disease

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
ORPHA:210272

**Comment by pnrobinson** [comment]
This sign is associated with this disease
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5764463/

**Comment by e-lbtn** [comment]
Absolutely, but we ask for the creation of this term because for us it's a specific sign of the "Mal de débarquement" and we need it to annotate and describe this disease in Orphanet

**Comment by pnrobinson** [comment]
I think the proper term would be 

Label: False perception of self-motion
Def: A perception that one's body is moving or swaying despite lack of motion of the body.
I am uncertain about the parent term
PMID:29443215

**Comment by pnrobinson** [comment]
I think the best parent is Vestibular dysfunction and have added this term.

**Comment by ohkokosnuss** [comment]
do you have matlab simulink code?
